src/components/Register.vue

```diff
@@ -0,0 +1,161 @@
+                        <input type="email" class="form-control" id="floatingInput" placeholder="name@example.com" v-model="email">
+                        <label for="floatingInput">Email Address</label>
+                    </div>
+                    
+                    <div class="form-floating">
+                        <input 
+                            type="password" 
+                            class="form-control" 
+                            id="floatingPassword" 
+                            placeholder="Password" 
+                            v-model="password" 
+                            :disabled="submitting">
+                        <label for="floatingPassword">Password</label>
+                    </div>
+                    
+                    <div class="form-floating">
+                        <input 
+                            type="password" 
+                            class="form-control" 
+                            id="passwordConfirmation" 
+                            placeholder="Password Confirmation" 
+                            v-model="password_confirmation" 
+                            :disabled="submitting">
+                        <label for="passwordConfirmation">Confirm Password</label>
+                    </div>
+                    
+                    <button class="w-100 btn btn-lg btn-primary" type="submit" :disabled="submitting">
+                        <div class="spinner-border text-light" role="status" v-if="submitting">
+                            <span class="visually-hidden">Loading...</span>
+                        </div>
+                        <span v-if="!submitting">Register</span>
+                    </button>
+
+                    <h6 class="mt-2 text-muted">If you already have an account?</h6>
+                    <p class="mt-2 mb-3 text-muted"><a href="/login">Login Here</a></p>
+                    <p class="mt-4 mb-3 text-muted">&copy; 2021 Pizza Hub, Inc</p>
+                </form>
+            </main>
+        </div>
+    </main>
+</template>
+
+<script >
+import api from '../utils/requests';
+
+export default {
+    data () {
+        return {
+            name: '',
+            email: '',
+            password: '',
+            password_confirmation: '',
+            submitting: false,
+            showSuccess: false,
+            showFailed: false
+        }
+    },
+    methods: {
+        handleSubmit(e) {
+            let currentObject = this;
+        
+            const response = api().post('/register', {
+                name: this.name, 
+                email: this.email, 
+                password: this.password,
+                password_confirmation: this.password_confirmation
+            }).then(function (response) {
+                const data = response.data;
+                console.log(data);
+                
+                if (data.status === 'success') {
+                    currentObject.showSuccess = true;
```

Comment:
Not sending to dashboard when success? No need to build out the email verfication. 😅

Reply:
This I agree, however, the logic when a client clicks the **proceed to checkout** button should be, they are redirected to log in if they have an account or register if they don't have an account, and once they are able to log in, they can view the **order summary** that is scrollable depending on the number of products within the order and also add **delivery details** then the order summary will have (cost of items and a delivery cost) and also a section of pay using (Mobile money, Visa or any other payment method which prompts them to add their mobile number, and confirm their order (Confirm your order button). That is how my thought process would be to actually complete the order until the client gets their product/pizza.

Reply:
Good suggeestion